### ISSUE TITLE
Graphical glitches in Tekken 2

### ISSUE THREAD

**Issue by Scallonni** [libretro/swanstation#130]
![Tekken 2 (USA)-240911-004531](https://github.com/user-attachments/assets/363817cc-51d0-49e6-8a7c-811fd9421da8)
Polygons stretch occasionally. This happens randomly for a few seconds and then stops shortly after. Maybe like a loop, it repeats every 30 seconds or more. It's very hard to detect, and one possible scenario to reproduce this defect clearly that I found is: playing with Law until stage 8 (the rival is always Baek) and pressing start, waiting for it to happen.
I've already checked all the core setup options in Retroarch, including completely disabling PGXP, testing many alternative setups to the default, ufff... Nothing worked.

**Comment by Awakened0** [comment]
Switching CPU Execution Mode to Interpreter should fix this.

**Comment by LibretroAdmin** [comment]
I can't reproduce it here. Is it because I have PGXP enabled? I will try another run without PGXP enabled. It might have already gotten fixed inadvertently in current master

**Comment by Awakened0** [comment]
I played for a bit with the latest nightly using the software renderer and the CPU Execution Mode set to Recompiler. I didn't manage to reproduce that problem this time.

**Comment by LibretroAdmin** [comment]
Yeah i think its gone. I will close the issue. If you still happen to reproduce it, write here and let me know, we can reopen it then
